cluster-service/Makefile

```diff
@@ -30,7 +32,12 @@ deploy: deploy-namespace-template deploy-istio-configurations-template ${DB_SECR
 	  -p AZURE_MI_MOCK_SERVICE_PRINCIPAL_PRINCIPAL_ID=${AZURE_MI_MOCK_SERVICE_PRINCIPAL_PRINCIPAL_ID} \
 	  -p AZURE_MI_MOCK_SERVICE_PRINCIPAL_CLIENT_ID=${AZURE_MI_MOCK_SERVICE_PRINCIPAL_CLIENT_ID} \
 	  -p AZURE_MI_MOCK_SERVICE_PRINCIPAL_CERT_NAME=${MI_MOCK_SERVICE_PRINCIPAL_CERT_NAME} \
-	  -p DATABASE_AUTH_METHOD=${DATABASE_AUTH_METHOD} | oc apply -f -
+	  -p DATABASE_AUTH_METHOD=${DATABASE_AUTH_METHOD} \
+	  -p DATABASE_AUTH_METHOD=${DATABASE_AUTH_METHOD} \
+	  -p AZURE_ARM_HELPER_IDENTITY_CLIENT_ID=${AZURE_ARM_HELPER_IDENTITY_CLIENT_ID} \
+	  -p AZURE_ARM_HELPER_IDENTITY_CERT_NAME=${MOCK_FPA_CERT_NAME} \
```

Comment:
```suggestion
	  -p AZURE_ARM_HELPER_IDENTITY_CERT_NAME=${ARM_HELPER_CERT_NAME} \
```

Reply:
as per the suggestion in https://github.com/Azure/ARO-HCP/pull/695/files#r1862360641

Reply:
fixed